c-pallets/segment-book/src/lib.rs

```diff
@@ -141,6 +144,7 @@ pub mod pallet {
 	pub type BoundedList<T> =
 		BoundedVec<BoundedVec<u8, <T as Config>::StringLimit>, <T as Config>::StringLimit>;
 
+	// Where does this number is coming from?
 	pub const LIMIT: u64 = 18446744073709551615;
```

Comment:
In fact, this is the maximum value of u64. I modified its expression and used `u64::Max`:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R141